### DISCUSSION TITLE
Kali linux - dbus-launch (no such file or directory)

### DISCUSSION THREAD

**Discussion by M4UR1-code** [microsoft/WSL#6502] [General]
Hello everyone
When I enter my username and password to enter the Kali Linux GUI in windows 10, through the remote, I get this error.
What should I do?
![No such file or directory](https://user-images.githubusercontent.com/77522364/105904087-b5177280-5fee-11eb-8fc9-368d86cd2aa5.jpeg)

**Comment by M4UR1-code** [comment]
Ya lo solucione,  se deben ingresar los siguientes comandos:

sudo service xrdp stop
sudo apt-get install dbus-x11
sudo service xrdp start

Gracias a mi mismo por tan diligente colaboración.  :-)

**Reply by kegoriEXE** [reply]
Hola, amigo, tengo el problema que me dice `E: package 'dbus-x11' has no istallation candidate` Agradeceria mcuho tu ayudo! es para un nethunter en un oppo find 5 lite. saludos

**Comment by codetomatot** [comment]
I have the same problem but using kali-win-kex so when i write kex it opens it. Any solutions?

**Comment by M4UR1-code** [comment]
Hello  hueychopper
I was researching kali-win-kex, but all the material that I find is the
same as the solution that I apply in my system, I share that youtube url
where you will find a tutorial, I hope it helps you.

https://youtu.be/Ppo0RL2f6pY

**Comment by codetomatot** [comment]
thanks

**Comment by oap5** [comment]
Thank you all very much! :)

**Comment by Flisu25** [comment]
What do i do on android when i try to open kali thru termux and the same thing happens?

**Reply by Aj0786** [reply]
https://youtu.be/SZRlY35k-pI?si=rKw1k0gEzkCH7VSz

**Comment by chiragkali** [comment]
No superuser and saying you are not rooted
When I typed this
sudo service xrdp stop
sudo apt-get install dbus-x11
sudo service xrdp start

**Comment by chiragkali** [comment]
Yes but It is not opening only  iam trying daily twice and trice
